### DISCUSSION TITLE
No data being created in DB via Prisma

### DISCUSSION THREAD

**Discussion by jcohenho** [planetscale/discussion#183] [Bug Reports] [answered]
I have a local Remix app configured with Prisma. I'm connected to Planetscale via `pscale connect mydb main --port 3309` and have my DATABASE_URL set to `mysql://root@127.0.0.1:3309/mydb`. I can successfully write data to the DB, but I don't see any logs in the planetscale CLI when this happens, and then in the PlanetScale dashboard I'm seeing `row writes  0 / 10 million` despite the fact that some data exists now in the DB. However, it does show 240kb of storage used. Is this normal behavior?

**Comment by jgreet** [comment, marked as answer]
> I don't see any logs in the planetscale CLI when this happens

That's normal, there's no output from the proxy when everything is working. If you enable `--debug` it will show new connections but it won't log queries.

Row read and write numbers aren't updated in real time, there should be a note below them that reads something like `Usage as of 42 minutes ago` If the writes are never added to the monthly total then there could be a problem.
